src/components/Admin/index.jsx

```diff
@@ -308,6 +309,9 @@ class Admin extends React.Component {
           <>
             <Helmet title="Learner Progress Report" />
             <Hero title="Learner Progress Report" />
+            <div className="mt-4">
+              <BudgetExpiryAlertAndModal />
+            </div>
```

Comment:
nit: if no alert is rendered, it appears (per the Admin.test.jsx.snap snapshot file) this renders an empty div with `mt-4`. Is this adding extra whitespace to the Learner Progress Report when there is no budget expiry alert?

Perhaps the `.mt-4` could be added within the `BudgetExpiryAlertAndModal` component directly, or have it rendered within the `div.col` below where we render the `<h2>` heading, which already has a `.mt-4` specified all the time.